### PR TITLE
addons: remove duplicated install code

### PR DESCRIPTION
**Pull request by MilhouseVH**
This is to eliminate duplicated code, functionality should remain largely unchanged.

@lrusak I've included the new `scripts/install_addon` script which is currently called only from `scripts/create_addon`. Maybe in future this can be expanded upon if we ever need to install addons during the main build.

### CODE REVIEW

File: scripts/install_addon [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  sed -e "/@PKG_ADDON_NEWS@/ \
+         {
+           r $PKG_DIR/changelog.txt
+           d
+         }" -i $ADDON_BUILD/$PKG_ADDON_ID/addon.xml
+else
+  sed -e "s|@PKG_ADDON_NEWS@||g" -i $ADDON_BUILD/$PKG_ADDON_ID/addon.xml
+fi
+
+PROVIDER_NAME="libreelec.tv"
```
Comment (chewitt) [inline scripts/install_addon]: Team LibreELEC

Comment (MilhouseVH) [inline scripts/install_addon]: Done.

**Comment by MilhouseVH** [comment]
Actually, functionality will be changed slightly by this change, hopefully for the better - all installed shared object libraries will now have the executable file permission applied (they don't at the moment, eg. `screensavers.rsxrs`) which will mean that `debug_strip()` will process them correctly as this function is only interested in executable files.

**Comment by MilhouseVH** [comment]
If no objections, could we get this in before it rots?

**Pull request merged**